### PR TITLE
Prevent "Admin Admin" from being in a page title

### PR DESCRIPTION
**Pull request by scottcwilson**
Check for the string "Admin" at the start of the name before adding it as a prefix.

### CODE REVIEW

**Comment by dakanji** [comment]
How does this impact non English set ups?

**Comment by scottcwilson** [comment]
It uses the define values, not the actual string.

File: admin/includes/init_includes/init_templates.php [hunk 1]
```diff
@@ -43,5 +43,9 @@
 if ($pagename == '') {
   $pagename = STORE_NAME;
 }
-$title = TEXT_ADMIN_TAB_PREFIX . ' ' . $pagename;
+if (strncmp(TEXT_ADMIN_TAB_PREFIX, $pagename, strlen(TEXT_ADMIN_TAB_PREFIX)) == 0) {
+   $title = $pagename;
+} else {
+   $title = TEXT_ADMIN_TAB_PREFIX . ' ' . $pagename;
+}
```
Comment (mc12345678) [inline admin/includes/init_includes/init_templates.php]: Isn't the current recommendation for if/else style code to keep the original line but if it meets the proposed condition reassign it?
I.e.
```diff
$title = TEXT_ADMIN_TAB_PREFIX . ' ' . $pagename;
+if (strncmp(TEXT_ADMIN_TAB_PREFIX, $pagename, strlen(TEXT_ADMIN_TAB_PREFIX)) == 0) {
+    $title = $pagename;
+}
```

**Comment by scottcwilson** [comment]
Generally that's a good policy but in this case, for clarity, I think the current logic is better and easier to read.  This style of titling based on page name is new for Zen Cart and people might want to change it.

File: admin/includes/init_includes/init_templates.php (hunk 1 above)
Comment (scottcwilson) [reply admin/includes/init_includes/init_templates.php]: If you got to the home page and look at the title in your browser tab you'll see it says "Admin Admin Home."  That was the driver for this change.

Comment (mc12345678) [reply admin/includes/init_includes/init_templates.php]: Ok, I do think I've seen it, but thought it "had always been that way". :)

**Comment by mc12345678** [comment]
Thought that in either code "style" that would be possible, I do somewhat like being able to have the additional information, though.

**Pull request merged**